pallets/schemas/src/lib.rs

```diff
@@ -143,6 +150,7 @@ pub mod pallet {
 
 	#[pallet::pallet]
 	#[pallet::generate_store(pub(super) trait Store)]
+	#[pallet::storage_version(SCHEMA_STORAGE_VERSION)]
```

Comment:
Storage version will be checked at the time of migration and anything older will be translated while newer ignored

Reply:
I'm curious how this works under the hood. Just want to make sure there are no surprises